guardian/prosemirror-embeds
# Add cypress for testing

## What does this change?

Adds Cypress for testing. Cypress will let us run unit and integration tests in a real DOM environment.

## How to test

- Run `yarn test`. Does Cypress appear, and run the tests when prompted?
- Run `yarn test-ci`. Does Cypress run all the tests, and produce the correct output?

There's **one problem** (arguably minor) that I can't figure out, yet – in interactive mode, you must manually refresh the tests, or change a file (the round arrow), to run them first time. I'm not sure why this is – any tips welcome!

## How can we measure success?

It's easy to add unit and integration tests to the project.
